.github/actions/package-arm/action.yml

```diff
@@ -0,0 +1,40 @@
+name: Package Rasberry Pi Release
+
+inputs:
+  arch:
+    required: false
+  target:
+    required: true
+  github_token:
+    required: true
+
+runs:
+  using: composite
+  steps:
+    - uses: ./.github/actions/setup-arm
+      with:
+        arch: ${{ inputs.arch }}
+        target: ${{ inputs.target }}
+
+    - name: Build
+      shell: bash
+      run: |
+        cargo build --release --all --target ${{ inputs.target }} --features=raspberry
+
+    - name: Compress
+      shell: bash
+      run: |
+        zip -j cargo-espflash-${{ inputs.target }}.zip target/${{ inputs.target }}/release/cargo-espflash
+        zip -j espflash-${{ inputs.target }}.zip target/${{ inputs.target }}/release/espflash
+
+    - uses: svenstaro/upload-release-action@v2
+      with:
+        repo_token: ${{ inputs.github_token }}
+        file: cargo-espflash-${{ inputs.target }}.zip
```

Comment:
Not sure if it's a good idea, but I'll throw it anyway, we can use `*.zip` here and avoid using the action twice.